src/Core/Types.Filters/Object/ObjectFilterFieldDescriptor.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using HotChocolate.Types.Descriptors;
+
+namespace HotChocolate.Types.Filters
+{
+    public class ObjectFilterFieldDescriptor
+       : FilterFieldDescriptorBase
+       , IObjectFilterFieldDescriptor
+    {
+        private readonly Type type;
+        public ObjectFilterFieldDescriptor(
+            IDescriptorContext context,
+            PropertyInfo property,
+            Type type)
```

Comment:
Should i remove type and just read the type over the property?

Reply:
I will have to check.